src/site/content/en/live/index.njk

```diff
@@ -30,16 +30,22 @@ pageScripts:
         <div class="w-event-section__column-left">
           <div class="w-event-heading">
             <h1 class="w-event-heading__logo">web.dev<span>/LIVE</span>
-              <video width="156" height="72" autoplay muted loop playsinline>
-                <source src="live.mp4" type="video/mp4" />
-                <img src="live.png" alt="live" />
-              </video>
+            {% Video
+              src="video/tcFciHGuF3MxnTr1y5ue01OGLBn2/vBhlsAXOy80qltOoAZ3J.mp4",
+              width="156",
+              height="72",
+              autoplay=true,
+              muted=true,
+              loop=true,
+              playsinline=true,
```

Comment:
it seems like `playsinline` isn't a standalone attribute but is instead part of autoplay
https://github.com/GoogleChrome/webdev-infra/blob/main/shortcodes/Video.js#L81

Reply:
Updated the library to support it.